guide/onboarding/protecting-a-wallet.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: guide
+title: Protecting a wallet
+description: Adding extra layers of security
+parent: Onboarding
+nav_order: 6
+permalink: /guide/onboarding/protecting-a-wallet/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+This page provides an overview to a few patterns that can be incorporated by wallets to hide users sensitive information.
+
+Illustration sources
+
+- https://www.figma.com/file/TGJ8VxZOQ65In8kCir6eQt/Hide-sensitive-information-illustration?node-id=0%3A1
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/protecting-a-wallet/hide-sensitive-information-header.png"
+   retina = "/assets/images/guide/onboarding/protecting-a-wallet/hide-sensitive-information-header.png"
+   mobile ="/assets/images/guide/onboarding/protecting-a-wallet/hide-sensitive-information-header.png"
+   mobileRetina ="/assets/images/guide/onboarding/protecting-a-wallet/hide-sensitive-information-header.png"
+   alt-text = "Payment Privacy & Security"
+   caption = 'Person by [DrawKit](https://www.drawkit.io/illustrations/mobile-article-colour){:target="_blank" rel="nofollow"} and trees by [Flaticon](https://www.flaticon.com/free-icon/tree-with-many-leaves_25267){:target="_blank" rel="nofollow"}'
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Protecting a wallet
+
+Bitcoin wallets contain sensitive information such as recovery phrases, balances, transaction histories, and more. Therefore, it is crucial to add extra layers of security to keep private data out of bad actors' hands. Additional protection may also be necessary for users who share devices, such as parents with children.
+
+A common practice for wallets is to have a barrier (i.e. Touch ID, PINs, etc.) that must be unlocked to open a wallet application. Creating robust and challenging- to- crack passwords and PIN codes isn't something that everyone excels at. In order to decrease the chance of someone’s wallet getting hacked because of poor security decisions, you can implement the following into your design:
```

Comment:
```suggestion
A common practice for wallets is to have a barrier (i.e., Touch ID, PINs, etc.) that must be unlocked to open a wallet application. Creating robust and challenging-to-crack passwords and PIN codes isn't something that everyone excels at. To decrease the chance of someone’s wallet getting hacked because of poor security decisions, you can implement the following into your design:
```